src/lang/KclSingleton.ts

```diff
@@ -738,6 +738,11 @@ export class KclManager {
     return ast.start === 0 && ast.end === 0 && ast.body.length === 0
   }
 
+  // Determines if there is no KCL code which means it is executing a blank KCL file
+  isAstBodyEmpty(ast: Node<Program>) {
+    return ast.body.length === 0
+  }
```

Comment:
Nit: do you mind updating the comment to make it clearer that this is different than the private method right above?

Reply:
Updated the comment, thanks!